aspnetcore/data/ef-mvc/intro.md

```diff
@@ -24,6 +24,9 @@ The sample application is a web site for a fictional Contoso University. It incl
 
 EF Core 1.1 is the latest version of EF but does not yet have all the features of EF 6.x. For information about how to choose between EF 6.x and EF Core 1.0, see [EF Core vs. EF6.x](https://docs.microsoft.com/ef/efcore-and-ef6/). If you choose EF 6.x, see [the previous version of this tutorial series](https://docs.microsoft.com/aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-an-entity-framework-data-model-for-an-asp-net-mvc-application).
 
+> [!NOTE]
+> For the Visual Studio 2015 version of this tutorial, see the [PDF of ASP.NET Core documentation as it was before the Visual Studio 2017 update](https://github.com/aspnet/Docs/blob/master/aspnetcore/common/_static/aspnet-core-project-json.pdf).
```

Comment:
That's accurate but clumsy. Can you make it less detailed but close enough? Something like
 
For VS15, see the [VS15 version of ASP.NET Core documentation in PDF format]
That's not great, but consider replacing "ASP.NET Core documentation as it was before the Visual Studio 2017 update" with  something like the VS15 version.